src/main/java/com/google/cloud/tools/managedcloudsdk/process/CollectingLineHandler.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.managedcloudsdk.process;
+
+/** A line handler that collects lines and returns a single result string. */
+public class CollectingLineHandler implements LineHandler<String> {
+
+  private final StringBuilder result = new StringBuilder("");
+
+  @Override
+  public void line(String line) {
+    if (line == null) {
+      return;
+    }
+    result.append(line).append(System.lineSeparator());
```

Comment:
depending on what this is used for a literal line break character might be better.

Reply:
makes sense.